entrypoint.sh

```diff
@@ -1,14 +1,14 @@
 #!/bin/sh
 set -e
 
-ELF="/app/malda_rs/bin/get-proof-data.bin"
-IDFILE="/app/malda_rs/src/elfs_ids.rs"
+ELF="/app/target/riscv-guest/methods/guests/riscv32im-risc0-zkvm-elf/release/get-proof-data.bin"
+METHODS_RS=$(find /app/target -name "methods.rs" -path "*/methods-*/out/*" | head -1)
 
 printf "================ RISC0 Guest ELF ================\n"
 printf "Path:      %s\n" "$ELF"
 printf "SHA256:    %s\n" $(sha256sum "$ELF" | cut -d' ' -f1)
 printf "Image ID:  "
-grep "_ID" "$IDFILE" | awk -F'= \\[|];' '{print "[" $2 "]"}'
+grep "_ID" "$METHODS_RS" | awk -F'= \\[|];' '{print "[" $2 "]"}'
```

Comment:
Current banner can show confusing output if files are missing (e.g., SHA256 blank, grep errors). 

Maybe add explicit `test -f "$ELF"` / `test -f "$METHODS_RS"` checks and fail with a clear message. 

Also, `grep "_ID"` is broad; prefer grepping the exact constant (e.g. `GET_PROOF_DATA_ID`) to avoid printing multiple IDs.

Reply:
Fixed in https://github.com/malda-protocol/malda-zk-coprocessor/commit/0a0d67e2a48dce54f6f1c2a26662b3f3452efa48.